src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -293,15 +291,18 @@ export const useSwapper = () => {
     })
   }
 
-  const setFees = async (trade: Trade<ChainTypes> | TradeQuote<ChainTypes>, sellAsset: Asset) => {
+  const setFees = async (
+    trade: Trade<SupportedChainIds> | TradeQuote<SupportedChainIds>,
+    sellAsset: Asset,
+  ) => {
     const feePrecision = feeAsset.precision
     const feeBN = bnOrZero(trade?.feeData?.fee).dividedBy(bn(10).exponentiatedBy(feePrecision))
     const fee = feeBN.toString()
 
-    switch (sellAsset.chain) {
-      case ChainTypes.Ethereum:
+    switch (sellAsset.chainId) {
+      case 'eip155:1':
```

Comment:
Do we really want to go from some typed and abstracted `ChainType` to magic strings?
See `ethChainId` and others in https://github.com/shapeshift/lib/pull/572

Reply:
💯 !